testapps/testapp/build.gradle

```diff
@@ -29,7 +29,7 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
-def msalVersion = "5.2.0"
+def msalVersion = "5.+"
```

Comment:
lets keep it to specific version.

Reply:
Talked with Amit offline; keeping these to +.